godot-codegen/src/util.rs

```diff
@@ -230,7 +236,17 @@ pub fn safe_ident(s: &str) -> Ident {
 fn to_hardcoded_rust_type(ty: &str) -> Option<&str> {
     let result = match ty {
         "int" => "i64",
+        "uint8_t" => "u8",
+        "uint16_t" => "u16",
+        "uint32_t" => "u32",
+        "uint64_t" => "u64",
+        "int8_t" => "i8",
+        "int16_t" => "i16",
+        "int32_t" => "i32",
+        "int64_t" => "i64",
         "float" => "f64",
+        "real_t" => "real",
+        "void" => "c_void",
         "String" => "GodotString",
         "Array" => "VariantArray",
         //"enum::Error" => "GodotError",
```

Comment:
Could you add a separate section with all the new mappings, that is introduced with a comment
```rs
...

// Types needed for native structures
...
```

Reply:
:+1: